parallel-install/example/components.yaml

```diff
@@ -0,0 +1,35 @@
+---
+defaultNamespace: kyma-system
+prerequisites:
+  - name: "cluster-essentials"
```

Comment:
the namespace is missing here is it delibrate ?

Reply:
Yes, that's intended. It will take the value from "defaultNamespace" it no namespace-field is provided.